docs/encyclopedia/nexus-operations.mdx

```diff
@@ -0,0 +1,184 @@
+
+## Nexus Operation Lifecycle {#operation-lifecycle}
+
+When you execute a Nexus Operation from a caller Workflow using the Temporal SDK, a command is sent to Temporal to schedule the Nexus Operation which is atomically handed off to the Nexus Machinery.
+The [Nexus Machinery](/glossary#nexus-machinery) uses state-machine-based invocation and completion callbacks to ensure [at-least-once](#at-least-once-execution-semantics-and-idempotency) execution of a Nexus Operation and reliable delivery of the result.
+
+![Nexus Overview](/img/nexus-overview.png)
+
+The Nexus Machinery is responsible for making the Nexus RPC calls on behalf of the caller Workflow, with [automatic-retries](#automatic-retries).
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with the Temporal Service.
+
+### System interactions
+
+Nexus Operations are often registered in the same Worker as the underlying Temporal primitives they abstract.
+
+![NexusYourCloud](/img/nexus/nexus-connect-queue-based-workers.png)
+
+For example, when you execute a Nexus Operation in a caller Workflow:
+1. **RespondWorkflowTaskCompleted([ScheduleNexusOperation command](/references/commands#schedulenexusoperation))** \-
+is used by the caller Worker to schedule a Nexus Operation, which atomically hands off execution to the caller's Nexus Machinery.
+2. **PollNexusTaskQueue** \-
+is used by the handler Worker to recieve a [Nexus Task](/workers#nexus-task) to process, for example to start a Nexus Operation.
+3. **RespondNexusTaskCompleted** or **RespondNexusTaskFailed** \-
+is used by the handler Worker to return the Nexus Task result.
+When asked to start a Nexus Operation, the Nexus handler decides if the Operation will be synchronous or asynchronous.
+   1. This is typically a static decision based on the [Temporal SDK builder function used](#sdk-support).
+   2. [Asynchronous Nexus Operations](#asynchronous-operation-lifecycle), created with the New-Workflow-Run-Operation SDK helper, will return a Nexus Operation ID, that can be used to perform additional actions like canceling an Operation.
+   3. [Synchronous Nexus Operations](#synchronous-operation-lifecycle), created with the New-Sync-Operation SDK helper, will return the Operation result directly.
+The caller's Nexus Machinery recieves the result and records a [NexusOperation event](/references/events#nexusoperationstarted) in the caller's Workflow History.
+4. **PollWorkflowTaskQueue** \-
+is used by the caller Worker to recieve a Workflow Task with the [Nexus Operation event](/references/events#nexusoperationstarted) which may be Started, Completed, Failed, Canceled, or TimedOut.
+
+### Asynchronous Operation lifecycle {#asynchronous-operation-lifecycle}
+
+An asynchronous Nexus Operation may take up to 60 days to complete in Temporal Cloud, to allow for callback URL token validation with asymmetric key rotation at those intervals.
```

Comment:
We weren't planning on using asymmetric keys FTR. Can we also move as much of the cloud specific content out of the concept docs? Don't need to make Cloud too much in users' faces IMHO.

Reply:
Fair point. Wanted to provide context on the 60 day limit, which seems arbitrary without the extra info, but will shorten it up.